monitoring/snippets/metric_delete.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package snippets
+
+// [START monitoring_delete_metric]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	monitoring "cloud.google.com/go/monitoring/apiv3"
+	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
+)
+
+// deleteMetric deletes the given metric. name should be of the form
+// "projects/PROJECT_ID/metricDescriptors/METRIC_TYPE".
+func deleteMetric(w io.Writer, name string) error {
+	ctx := context.Background()
+	c, err := monitoring.NewMetricClient(ctx)
+	if err != nil {
+		return err
+	}
+	req := &monitoringpb.DeleteMetricDescriptorRequest{
+		Name: name,
+	}
+	err = c.DeleteMetricDescriptor(ctx, req)
```

Comment:
I kind of like the old, unexpanded version. Should this be re-condensed into `if err = c.DeleteMetricDescriptor(ctx, req); err != nil {` ?

Reply:
Agreed. Done.